package.json

```diff
@@ -91,6 +91,7 @@
     "react-html-parser": "^2.0.2",
     "react-router": "3",
     "rimraf": "^3.0.2",
-    "sinon": "^9.2.2"
+    "sinon": "^9.2.2",
+    "web-components": "https://github.com/department-of-veterans-affairs/component-library.git#wc-v0.2.0"
```

Comment:
This will add the release `wc-v0.2.0` which is based off of the `web-components` branch.